packages/nimbus/src/components/money-input/money-input.tsx

```diff
@@ -361,16 +342,17 @@ export const MoneyInputComponent = (props: MoneyInputProps) => {
 
       {/* High Precision Badge */}
       {hasHighPrecisionBadge && isCurrentlyHighPrecision && (
-        <MoneyInputBadgeSlot data-testid="high-precision-badge">
+        <MoneyInputBadgeSlot>
```

Comment:
This could probably wrap the `Box` directly if it ends up being necessary after `trailingIcon` is implemented? idk